src/360-core/browzine-360-core-adapter.js

```diff
@@ -191,7 +200,7 @@ browzine.serSol360Core = (function() {
 
     var template = "<div class='browzine' style='margin: 5px 0;'>" +
                      "<img alt='BrowZine Book Icon' class='browzine-book-icon' src='{bookIcon}' style='margin-top: -3px; display: inline;' width='16' height='15'/> " +
-                     "<a class='browzine-web-link' href='{browzineWebLink}' target='_blank' style='font-weight: 300;'>{browzineWebLinkText}</a>" +
+                     "<a class='browzine-web-link' href='{browzineWebLink}' target='_blank' style='font-weight: 300;' onclick='browzine.serSol360Core.transition(event, this)'>{browzineWebLinkText}</a>" +
```

Comment:
👍 to adding this behavior to all 3 services we support for maximum future-proofing! 

It'll involve a little extra testing now, but would save hours in the future if a change ever makes this necessary.